solr/contrib/jaegertracer-configurator/src/test/org/apache/solr/jaeger/TestJaegerConfigurator.java

```diff
@@ -53,26 +51,23 @@ public void doBefore() {
 
   @Test
   public void testInjected() throws Exception {
-    MiniSolrCloudCluster cluster = new SolrCloudTestCase.Builder(2, createTempDir())
-        .addConfig("config", TEST_PATH().resolve("collection1").resolve("conf"))
-        .withSolrXml(getFile("solr/solr.xml").toPath())
-        .build();
+    MiniSolrCloudCluster cluster =
+        new SolrCloudTestCase.Builder(2, createTempDir())
+            .addConfig("config", TEST_PATH().resolve("collection1").resolve("conf"))
+            .withSolrXml(getFile("solr/solr.xml").toPath())
+            .build();
     try {
       TimeOut timeOut = new TimeOut(2, TimeUnit.MINUTES, TimeSource.NANO_TIME);
       timeOut.waitFor(
           "Waiting for GlobalTracer is registered",
           () -> GlobalTracer.get().toString().contains("JaegerTracer"));
 
-      //TODO add run Jaeger through Docker and verify spans available after run these commands
+      // TODO add run Jaeger through Docker and verify spans available after run these commands
       CollectionAdminRequest.createCollection("test", 2, 1).process(cluster.getSolrClient());
-      new UpdateRequest()
-          .add("id", "1")
-          .add("id", "2")
-          .process(cluster.getSolrClient(), "test");
+      new UpdateRequest().add("id", "1").add("id", "2").process(cluster.getSolrClient(), "test");
```

Comment:
I'm not sure I agree with this reformatting. How are the rules configured for overrides? This is a minor nit, and if it's the only issue then I'll deal with all of it on one line, but I'd like to understand more about the plugin we're applying.

Reply:
> ... How are the rules configured for overrides? ... I'd like to understand more about the plugin we're applying.

From what I understand https://github.com/apache/solr/blob/a9a8d2023de00474277c6edbaff75d8bd12f33e8/gradle/validation/spotless.gradle#L37-L41 is the area where the configuration happens and as per https://github.com/diffplug/spotless/tree/main/plugin-gradle there are many options available (though from a little bit of "browsing around" i couldn't find one that obviously covers the situation here).

Oh and please note that `spotless.gradle` is also being changed as part of this pull request (to remove the exemption for `contrib/jaegertracer-configurator`) but the github UI wouldn't let me add a comment on the relevant lines, hence commenting here where the question cropped up.